server/events/command_runner.go

```diff
@@ -254,7 +254,7 @@ func (c *DefaultCommandRunner) RunPRReviewCommand(ctx context.Context, repo mode
 		Pull:              pull,
 		PullStatus:        status,
 		HeadRepo:          repo,
-		Trigger:           command.AutoTrigger,
+		Trigger:           command.PRReviewTrigger,
```

Comment:
should be safe to change as field is only used by plan command runner at the moment.